Frontend/src/components/notificationsWidget/notificationsWidget.tsx

```diff
@@ -50,8 +51,8 @@ const Notification: React.FC<NotificationProps> = ({officialsId, notification})
   const [isCollapsed, setIsCollapsed] = React.useState(true)
   const collapse = (): void => setIsCollapsed(!isCollapsed)
 
-  const date = Intl.DateTimeFormat('cs-CZ').format(new Date(notification.FromDate))
-  const formatedDate = !notification.FromDate ? 'neznámé období' : date
+  const date = notification.FromDate && dummyFormatDateShort(new Date(notification.FromDate))
+  const formatedDate = notification.FromDate ? date : 'neznámé období'
```

Comment:
`const date = notification.FromDate ? dummyFormatDateShort(new Date(notification.FromDate)) : 'neznámé období'`

nebude to stacit?